erpnext/setup/doctype/quotation_lost_reason_detail/quotation_lost_reason_detail.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
Add Patch to update data from `Lost Reason Detail` to `Quotation Lost Reason Detail` for existing records
Also since now we have a separate doctype for `Quotation` maybe `Lost Reason Detail` can be renamed to `Opportunity Lost Reason Detail` to reduce confusion